test/testunusedvar.cpp

```diff
@@ -1962,6 +1963,17 @@ class TestUnusedVar : public TestFixture {
                       errout_str());
     }
 
+    void structmember26() {
+        checkStructMemberUsage("struct S {\n"
```

Comment:
Either there are typos in the test case (`s` -> `S`), or I don't understand the FP.
As it stands, I would expect a warning for the given code.